internal/sql/repository/chartConfig/PipelineConfigRepository.go

```diff
@@ -114,8 +123,12 @@ func (impl PipelineConfigRepositoryImpl) GetDefaultStrategyByPipelineId(pipeline
 	return pipelineStrategy, err
 }
 
-func (impl PipelineConfigRepositoryImpl) Delete(pipelineStrategy *PipelineStrategy, tx *pg.Tx) error {
-	return tx.Delete(pipelineStrategy)
+// Delete -
+// it will soft-delete the pipeline strategy from the database
+func (impl PipelineConfigRepositoryImpl) Delete(pipelineStrategy *PipelineStrategy, userId int32, tx *pg.Tx) error {
```

Comment:
we have in past used method name as something like MarkAsDeleted, please use the same